dist/segmented-buttons/segmented-buttons.css

```diff
@@ -0,0 +1,76 @@
+.segmented-buttons {
+  border: 1px solid var(--color-stroke-default);
+  border-radius: 24px;
+  max-width: 720px;
+  min-width: 215px;
+  padding: 4px;
+}
+.segmented-buttons > ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+.segmented-buttons > ul > li {
+  display: flex;
+  flex: 1;
+}
+.segmented-buttons > ul > li:not(:first-child) {
+  margin-left: 8px;
+}
+.segmented-buttons__button {
+  background-color: transparent;
+  border: none;
+  border-radius: var(--btn-border-radius, calc(40px / 2));
+  font-size: 0.875rem;
+  min-height: 40px;
+  padding: 8px 16px;
+  width: 100%;
+}
+.segmented-buttons__button-cell {
+  align-items: center;
+  display: flex;
+  width: 100%;
+  justify-content: center;
+}
+.segmented-buttons__button-cell--fixed-height {
```

Comment:
Where do these fixed-height classes get used?

Reply:
Those are from the button mixins

Reply:
Hmm, might be worth refactoring the mixin then if it's not too much trouble. Otherwise everything else looks great!

Reply:
Okay, I went ahead and split it up. Wasn't too bad.